### ISSUE TITLE
How did you choose parameters?

### ISSUE THREAD

**Issue by kulkarni-vishwanath** [chetanambi/Food-Quality-Assessment#1]
Hi Chetan,

First of all, very useful notebook, learnt a lot about feature aggregation. I have one question, when you ran the Lightgbm model, how did you decide on the Hyperparameter values? Are these random set of Hyperparameters or did you finalize them after using a search parameter like Randomizedsearch or Gridsearch? It will be helpful if you could explain the method to choose these hyperparameters.

Thanks

**Comment by chetanambi** [comment]
Hi Vishwanath - thank you !

Regarding Hyperparameters, I always finalize them manually. Randomized or
Gridsearch are very computationally expensive and takes lot of time. As you
know, for many algorithms learning rate, depth, num_leaves etc. So i start
with main parameters of that algorithm and try to tune them first and then
later introduce other parameters.

For LightGBM, I start with learning rate, num_iterations, depth and
num_leaves and try to tune them manually. Then will use other parameters
later. This technique worked great for me till now.
